### DISCUSSION TITLE
RDF4J 4.0.0

### DISCUSSION THREAD

**Discussion by abrokenjester** [eclipse-rdf4j/rdf4j#3832] [Announcements]
RDF4J 4.0.0 is a major release of the Eclipse RDF4J framework. Some highlights:

-  Update to Java 11 as the minimally-required version of Java
-  An (experimental) new embedded triplestore: the [LMDB store](https://rdf4j.org/documentation/programming/lmdb-store)
- Improved support for working with RDF4J in Spring and Spring Boot applications (see [documentation](https://rdf4j.org/documentation/programming/spring) for details)
- Performance improvements in the SHACL engine
- Performance improvements in the Memory Sail
- Various speed and optimization improvements in the SPARQL query engine
- Various improvements in the SparqlBuilder
- SeRQL and SPIN are no longer supported
- Removal of several long-deprecated packages and interfaces

For more details, including instruction on how to upgrade, see the [release notes](https://rdf4j.org/release-notes/4.0.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/eclipse/rdf4j/releases/tag/4.0.0'>RDF4J 4.0.0</a>.</em>

**Comment by kenwenzel** [comment]
@jeenbroekstra The referenced LMDB documentation is not the latest version. Could you upload the latest documentation?

**Reply by abrokenjester** [reply]
Thanks for the heads up. Looks like we forgot to merge your documentation PR before release. Merged to main now, it should automatically appear on the site in the next 20 minutes or so.
